packages/typography/src/Link/Link.tsx

```diff
@@ -50,16 +51,21 @@ const Link = Polymorphic<LinkProps>(
 
     const baseFontSize = useUpdatedBaseFontSize(baseFontSizeOverride);
 
-    let target, icon, rel;
+    let target, rel, icon;
 
-    if (targetProp) {
+    // Respect passed values
+    if (targetProp || relProp) {
       target = targetProp;
+      rel = relProp;
     } else {
-      if (hrefHostname === currentHostname) {
-        target = '_self';
-      } else {
-        target = '_blank';
-        rel = 'noopener noreferrer';
+      // Only manually set values if not provided and we are rendering an anchor tag
+      if (Component === 'a') {
```

Comment:
I'm assuming this is also inferring that usePolymorphic is returning `a` as the Component href is set in props

Reply:
confirmed